reactnative/RNPlayground/app/navigators/PlaygroundNavigator.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from "react"
+import { BaseScreenProps, NavigatorParamList } from "."
+import { createNativeStackNavigator } from "@react-navigation/native-stack"
+import { StackScreenProps } from "@react-navigation/stack"
+import { PhotosScreen } from "../screens/PhotosScreen"
+import { CompositeScreenProps } from "@react-navigation/native"
+import { PlaygroundScreen } from "../screens"
+import { Appbar } from "react-native-paper"
+import { colors } from "../theme"
+import { View } from "react-native"
+import { Text } from "../components"
+import { getHeaderTitle } from "@react-navigation/elements"
+import { translate } from "../i18n"
+import { Screen } from "../components"
+
+type PlaygroundTabParamList = {
+  Playground: undefined,
+  Photos: undefined
+}
+
+/**
+ * Helper for automatically generating navigation prop types for each route.
+ *
+ * More info: https://reactnavigation.org/docs/typescript/#organizing-types
+ */
+export type PlaygroundTabScreenProps<T extends keyof PlaygroundTabParamList> = CompositeScreenProps<
```

Comment:
I copied this approach from ignite demo code, not 100% how it works though https://github.com/infinitered/ignite/blob/master/boilerplate/app/navigators/DemoNavigator.tsx#L25